virtcontainers/sandbox.go

```diff
@@ -639,11 +640,17 @@ func (s *Sandbox) createCgroupManager() error {
 		cgroupPath = spec.Linux.CgroupsPath
 
 		// Kata relies on the cgroup parent created and configured by the container
-		// engine, but sometimes the sandbox cgroup is not configured and the container
-		// may have access to all the resources, hence the runtime must constrain the
-		// sandbox and update the list of devices with the devices hotplugged in the
-		// hypervisor.
-		resources = *spec.Linux.Resources
+		// engine by default. The exception is for devices whitelist as well as sandbox-level
+		// CPUSet.
+		resources.Devices = spec.Linux.Resources.Devices
```

Comment:
okay, I understand why cpu and memory are not being honoured, but what about blockIO ?

Reply:
There are a couple issues open for this ability now, and there's work in progress on K8S side to add this. Are you okay with holding off until that discussion (both on Kata and k8s side) complete to address this subsystem?